packages/esm-form-entry-app/src/app/app.module.ts

```diff
@@ -19,6 +19,9 @@ import { SingleSpaPropsService } from './single-spa-props/single-spa-props.servi
 import { FormCreationService } from './form-creation/form-creation.service';
 import { TranslateModule, TranslateService, TranslateStore } from '@ngx-translate/core';
 import { ProgramResourceService } from './openmrs-api/program-resource.service';
+import { registerLocaleData } from '@angular/common';
+import localeFr from '@angular/common/locales/fr';
+registerLocaleData(localeFr);
```

Comment:
So, this unconditionally loads the French locale as part of the application. I don't think we can accept this into the community version for that reason. Inter alia, for instances not using French and where French is not a display language, we wouldn't want to load this. Also, for locations which are not in French but want a different date format, this won't work.

Instead, we could support this by doing something similar around [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/02e38dbf3a6060e3ed795735b35917e0184b8970/packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts#L119C1-L119C1) (the point where we register the form language). Basically, we should be able to do something like:

```js
import(`@angular/common/locales/${language}`).then(registerLocaleData).catch(console.error);
```

The above might need some modifications to work with the RxJS chains.